include/ck/tensor_operation/gpu/device/impl/device_contraction_utils.hpp

```diff
@@ -0,0 +1,80 @@
+// SPDX-License-Identifier: MIT
```

Comment:
This new file is not installed by `make install`

Reply:
We have some other utils in this directory and it seems to work just fine.
I tested it by running `make install` in CK and I see:
`-- Installing: /opt/rocm/include/ck/tensor_operation/gpu/device/impl/device_contraction_utils.hpp`
in the log and the file is indeed there.